CI/ios/before_install.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+echo DEVELOPER_DIR=/Applications/Xcode_13.4.1.app >> $GITHUB_ENV
+
+curl -L 'https://github.com/kambala-decapitator/vcmi-ios-depends/releases/latest/download/vcmi-ios-depends-xc13.2.1.txz' \
```

Comment:
That's probably not a good idea

Reply:
dependencies are built by GitHub runners as well, don't see a reason to perform double work for nothing. At a later point this can be changed to build deps as part of the app build directly and use GH Actions caching mechanism.

Reply:
in the future I plan to try using Vcpkg or Conan to build all deps

Reply:
As for me that's the only one item needs to be fixed prior merge

Reply:
so what is your suggestion? Go Conan-only?

Reply:
Conan is the best option, alternative could be creating public repository with deps inside vcmi

Reply:
then I should simply move https://github.com/kambala-decapitator/vcmi-ios-depends to VCMI org :)

Conan can follow later.

Reply:
switched deps to VCMI repo